packages/evm/src/evm.ts

```diff
@@ -175,6 +175,9 @@ export class EVM implements EVMInterface {
     Hardfork.Paris,
     Hardfork.Shanghai,
     Hardfork.Cancun,
+    // Optimism (Bedrock) hardforks
+    Hardfork.Bedrock,
+    Hardfork.Regolith,
```

Comment:
Per my comments above, rather than this hardcoded list of supported hardforks, I'd prefer to see a path where we take in a list of supported hardforks from `common` (and let people add arbitrary ones there).  That way, it's not on us to maintain the list of hardforks for whatever L2 is popular next.